thrift0.13/src/main/java/com/linecorp/armeria/internal/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -495,11 +494,12 @@ public Map<String, String> loadDocStrings(Set<ServiceConfig> serviceConfigs) {
                              .flatMap(entry -> entry.interfaces().stream().map(Class::getClassLoader))
                              .flatMap(loader -> docstringExtractor.getAllDocStrings(loader)
                                                                   .entrySet().stream())
-                             .collect(toImmutableMap(Map.Entry::getKey, Map.Entry::getValue, (a, b) -> a));
+                             .collect(toImmutableMap(Map.Entry<String, String>::getKey,
+                                                     (Map.Entry<String, String> entry) ->
+                                                             DescriptionInfo.of(entry.getValue()),
+                                                     (a, b) -> a));
```

Comment:
Thanks ! I'll check it 😎

Reply:
`Functions.first()` return type is `public static <T, U> BiFunction<T, U, T>`, 
But toImmutableMap only supports `public static <T> BinaryOperator<T>`.

So It can't be used and It should be changed to be used.
```
// AS IS
public static <T, U> BiFunction<T, U, T> first() {
        return (first, second) -> first;
    }

// TO BE
public static <T> BinaryOperator<T> first2() {
    return (first, second) -> first;
}
```

Reply:
I don't know that it uses `BinaryOperator`. Please ignore the comment.